lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -266,6 +266,9 @@ protected function transformCampaign($data) {
         $output['uri'] = $data->uri;
       }
 
+      if (dosomething_helpers_get_variable('node', $data->id, 'magic_link_copy')) {
```

Comment:
probably need an `isset` here to avoid warnings? then you don't have to re-get the variable on the line below as well